cmd/goal/clerk.go

```diff
@@ -134,8 +133,7 @@ func init() {
 	compileCmd.Flags().StringVarP(&account, "account", "a", "", "Account address to sign the program (If not specified, uses default account)")
 
 	dryrunCmd.Flags().StringVarP(&txFilename, "txfile", "t", "", "transaction or transaction-group to test")
-	dryrunCmd.Flags().Uint64VarP(&round, "round", "r", 1, "round number to simulate")
-	dryrunCmd.Flags().Int64VarP(&timeStamp, "time-stamp", "S", 0, "unix time stamp simulate (default now)")
+	dryrunCmd.Flags().Int64VarP(&timeStamp, "time-stamp", "S", 0, "value for txn FirstValidTime (default now)")
```

Comment:
nit: mention it's the unix epoch time; I know it's obvious for you, but it might not be obvious for someone who uses the goal command-line.